### PR TITLE
[#1820] remove left margin for beta entry page buttons caused by #1793

### PR DESCRIPTION
**Pull request by kareila**
I'll go ahead and get this live if it gets the thumbs up.

### CODE REVIEW

**Comment by kareila** [comment]
Oh, yeah: fixes #1820.

File: htdocs/scss/pages/entry/new.scss [hunk 1]
```diff
@@ -75,6 +75,7 @@
 
     li {
         display: inline-block;
+        margin-left: 0 !important;
```
Comment (zorkian) [inline htdocs/scss/pages/entry/new.scss]: My understanding of CSS is that `!important` is a hack that shouldn't be used, or is there legitimate reasons to use it?

Comment (kareila) [inline htdocs/scss/pages/entry/new.scss]: It's used for things that should never be overridden by more specific targets, so it's appropriate here.  There are other lines in the same SCSS file that also use it.

**Comment by zorkian** [comment]
Thanks for the explanation!

**Pull request merged**